app/rpc/RPC.ts

```diff
@@ -1075,6 +1075,12 @@ export default class RPC {
         const progress = await JSON.parse(pro);
         const sendId = progress.id;
 
+        // bacause I don't know what the user are doing, I force every 2 seconds
+        // the interrupt flag to true if the sync_interrupt is false
+        if (!progress.sync_interrupt) {
+          await RPC.rpc_setInterruptSyncAfterBatch('true');
```

Comment:
Wow!

Reply:
Maybe never happen, but if this flag become `false`, I will change to `true` right away... (only in the time window when the App is sending)